app/src/main/java/com/iron/espresso/presentation/place/SearchPlaceDetailActivity.kt

```diff
@@ -45,107 +44,62 @@ class SearchPlaceDetailActivity : FragmentActivity(), OnMapReadyCallback {
     @UiThread
     override fun onMapReady(naverMap: NaverMap) {
 
-        var firstEvent = true
-        var firstAnim = false
+        var gesturing = false
```

Comment:
```suggestion
        var isGesuturing = false
```
요런 네이밍이 좀 더 일반적이지 않을까 생각해봅니다.

Reply:
네이밍 수정하였습니다